src/linking/zero_initializer.h

```diff
@@ -24,10 +24,21 @@ exprt zero_initializer(
   const namespacet &,
   message_handlert &);
 
+exprt nondet_initializer(
```

Comment:
I doubt that `linking` is the right place for this file. Maybe move to `goto-programs`?

Reply:
With some further fixes it seemed that `util` was actually a suitable place - moved.